docs/get-started/getting-started-on-prem.md

```diff
@@ -134,7 +134,7 @@ Now for each worker node:
   - `cp $SPARK_HOME/conf/spark-env.sh.template $SPARK_HOME/conf/spark-env.sh`
   - Edit `$SPARK_HOME/conf/spark-env.sh` and add any worker options. The example below sets the
     number of GPUs per worker to 4 and points to the discovery script.  Change this for your setup.
```

Comment:
this doesn't match below now.  we can leave this 4 or if you think its confusing change the text.

Reply:
I was unsure of whether to change that part of the example or not.  It was a little confusing to me, but I think that was just me :-)  I will change it back to 4.